### ISSUE TITLE
`extend #{$variable};` parse error in SCSS

### ISSUE THREAD

**Issue by bassettsj** [tonyganch/gonzales-pe#61]
Gonzales is having a hard time parsing an `@extend` with string interpolation for SCSS, see below.

```
parsing error: Please check the validity of the block starting from line #284

283 | .button-close {
284*|     @extend #{$button-close-icon} !optional;
286 |     &.ng-hide-remove,

Gonzales PE version: 3.0.0-26
```
### Related
- #31 
- #60

**Comment by kyranjamie** [comment]
+1

Current solution is to ignore affected files from the glob with `!` prefix (when using gulp plugin)

**Comment by varya** [comment]
+1 Suffer from this too

**Comment by tonyganch** [comment]
Should be fixed in `dev`, available via `3.0.0-27`
